main.js

```diff
@@ -115,33 +115,21 @@ Tracking.prototype.init = function(config) {
 
 	const hasDeclarativeConfig = !!this._getDeclarativeConfigElement();
 
-	if (!(hasDeclarativeConfig || config)) {
```

Comment:
I moved this to far lower - as far as I can tell, if no config is supplied there's no reason not so still set up the class correctly with version and required helpers.